main_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestCreatePhoneBook(t *testing.T) {
```

Comment:
You are missing the following tests:

* a case with multiple names without duplicates
* a case with multiple names with duplicates (>=3 duplicates)